charts/rancher-monitoring/rancher-monitoring/100.0.0+up16.6.0/Chart.yaml

```diff
@@ -15,6 +15,7 @@ annotations:
   catalog.cattle.io/requests-memory: 4000Mi
   catalog.cattle.io/type: cluster-tool
   catalog.cattle.io/ui-component: monitoring
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.22.0'
```

Comment:
I don't believe Monitoring V2 has a specific constraint for deploying in 1.22 clusters... afaik v2 should be fine. cc: @jiaqiluo since you're more familiar with the current status of Monitoring

Reply:
If so, we should change this to `catalog.cattle.io/kube-version: '>= 1.16.0-0'` just like alerting drivers

Reply:
@aiyengar2  is right, monitoring v2 should be able to run in 1.22 clusters.